eva-stats/stats_variant_warehouse.py

```diff
@@ -31,8 +31,13 @@ def get_database_name_from_assembly(metadata_handle, assembly):
             "from assembly a " \
             "left join taxonomy t on (t.taxonomy_id = a.taxonomy_id)" \
             "where assembly_accession = '{0}'".format(assembly)
-    result = get_all_results_for_query(metadata_handle, query)
-    database_name = 'eva_{0}_{1}'.format(result[0][0], result[0][1])
+    rows = get_all_results_for_query(metadata_handle, query)
+    if len(rows) == 0:
+        raise ValueError(f'No database for {assembly} found')
+    elif len(rows) > 1:
+        options = ', '.join((f'{r[0]}_{r[1]}' for r in rows))
+        raise ValueError(f'More than one possible database for {assembly} found: {options}')
+    database_name = f'eva_{rows[0][0]}_{rows[0][1]}'
```

Comment:
This looks good for now... But this and April's being long-lived scripts, it might be better if the functionality is extracted to a common library in the future and both of them using that function. Let's discuss more in the stand-up...

Reply:
Added to pyutils in [this PR](https://github.com/EBIvariation/eva-common-pyutils/pull/13) and used here